mesa: update to 25.3.x

MingcongBai opened the pull request:
Topic Description
-----------------

- mesa+32: update to 25.3.1
    - Disable assert\(\) for stable-bound updates, end-users can live without
    assert\(\).
    - Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.1
    \(HEAD: bd2eda4cfbcff30ae78598bc8933fde83dc31740\).
- mesa: update to 25.3.1
    - Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.1
    \(HEAD: bd2eda4cfbcff30ae78598bc8933fde83dc31740\).
- mesa+32: update to 25.3.0
- mesa: update to 25.3.0
- mesa+32: update to 25.3.0-rc4
    Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.0-rc4
    \(HEAD: 7ce39bc59cd9157b6aeec6796e5f85edbd831c47\).
- directx-headers+32: new, 1.618.2
- mesa: update to 25.3.0-rc4
    Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.0-rc4
    \(HEAD: 7ce39bc59cd9157b6aeec6796e5f85edbd831c47\).

Package(s) Affected
-------------------

- mesa: 1:25.3.1

Security Update?
----------------

No

Build Order
-----------

```
#buildit mesa
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`
- [x] LoongArch 64-bit (No SIMD) `loongarch64_nosimd`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

MingcongBai committed:
mesa: update to 25.3.0-rc4

Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.0-rc4
(HEAD: 7ce39bc59cd9157b6aeec6796e5f85edbd831c47).

MingcongBai committed:
directx-headers+32: new, 1.618.2

MingcongBai committed:
mesa+32: update to 25.3.0-rc4

Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.0-rc4
(HEAD: 7ce39bc59cd9157b6aeec6796e5f85edbd831c47).

MingcongBai commented:
@aosc-buildit-bot dickens

xtexx commented:
<img width="927" height="125" alt="image" src="https://github.com/user-attachments/assets/e566a1d9-0fc4-46f0-ae51-2eaeb0335b45" />

xry111 committed:
mesa: update to 25.3.0

xry111 committed:
mesa+32: update to 25.3.0

xry111 reviewed (dismissed):
It's easier to just wait for the merge of #13707.

HmnSn committed:
mesa: update to 25.3.1

- Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.1
  (HEAD: bd2eda4cfbcff30ae78598bc8933fde83dc31740).

HmnSn committed:
mesa+32: update to 25.3.1

- Disable assert() for stable-bound updates, end-users can live without
  assert().
- Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.1
  (HEAD: bd2eda4cfbcff30ae78598bc8933fde83dc31740).

MingcongBai committed:
mesa: update to 25.3.2

Track patches at AOSC-Tracking/mesa @ aosc/mesa-25.3.2
(HEAD: ec8fc2c482ecaad301338962c2ad9eca26b3cb2b).

MingcongBai commented:
> It's easier to just wait for the merge of #13707.

Merged.

MingcongBai committed:
libva+32: rebuild to correct repository dependency
